subsys/bluetooth/audio/Kconfig.bap

```
@@ -53,6 +53,13 @@ config BT_AUDIO_CODEC_CFG_MAX_METADATA_SIZE
 	help
 	  Number of octets to support for Codec Specific Configuration metadata.
 
+config BT_BAP_BASS_MAX_SUBGROUPS
+	int "BASE Subgroup Maximum Number of Subgroups support"
+	default 1
+	range 0 31
```

```suggestion
	range 0 25
```

Due limitations in advertising data, the maximum size of a BASE is 253, the maximum size of all subgroups are 249 and the minimum size of a subgroup is 10 octets, so effectively there can be a maximum of 24 subgroups in a BASE
```suggestion
	range 0 24
```